apps/src/studio/studio.js

```diff
@@ -1233,20 +1235,17 @@ function handleActorCollisionsWithCollidableList (
             // NOTE: if items are allowed to move outOfBounds(), this may never
             // be called because the last item may not be removed here.
 
-            if (list.length === 1) {
+            if (activeCollidables.length === 1) {
               callHandler('whenGetAllItems');
               Studio.trackedBehavior.gotAllItems = true;
             }
 
-            var className = collidable.className;
-            var itemCount = 0;
-            for (var j = 0; j < list.length; j++) {
-              if (className === list[j].className) {
-                itemCount++;
```

Comment:
You can see the same problem here where we count the items of a particular class - items currently fading out get counted.